From: Thomas Moulard <thomas.moulard@gmail.com>
Date: Tue, 11 Jun 2013 18:36:17 +0900
Subject: Fix plugindir for multiarch.

---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 701fdc7..ceb288b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -90,7 +90,7 @@ SETUP_PROJECT()
 
 # Add extra keys to pkg-config file to export the location of the
 # plug-in directory in a robust manner.
-SET(PKG_CONFIG_EXTRA "plugindir=${CMAKE_INSTALL_PREFIX}/lib/roboptim-core")
+SET(PKG_CONFIG_EXTRA "plugindir=${CMAKE_INSTALL_LIBDIR}/roboptim-core")
 
 # Add main library to pkg-config file.
 PKG_CONFIG_APPEND_LIBS(roboptim-core)
